src/routes/+page.svelte

```diff
@@ -32,6 +32,8 @@
 	}[] = [];
 	let solutionsSections: { title: string; description: string }[] = [];
 	let solutions: typeof solutionsSections = [];
+	let valuesSections: { icon: typeof SvelteComponent<SvelteHTMLElements["svg"]>; title: string; description: string }[] = [];
+	let values: typeof valuesSections = [];
```

Comment:
You actually don't need to have 2 variables here; I did that for the process because I needed to add/remove items dynamically in responsive mode, which you don't have to do here: `valuesSections` is enough! ;)

Reply:
Oh okay, my bad 😅